pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,28 +13,55 @@ node('jenkins-master') {
 }
 
 notify.wrap {
+  def retries = 3
+  def repo = 'webserv/imgserv'
+  def tag = 'dax_latest'
+
+  def run = {
+    dir('imgserv/lsst-dm-ci') {
+        stage('build') {
+            retry(retries) {
+                build(repo)
+            }
+        } // build
+
+        stage('test') {
+            retry(retries) {
+                docker.image(repo:tag).inside {
+                    util.bash "./run_tests.sh"
+                }
+            }
+        } // test
+
+        stage('publish') {
+            util.bash "./pub_image.sh"
```

Comment:
Is there a missing dir block here?

Reply:
I was trying to encapsulate all 3 stages in a single dir block - not the right syntax?

Reply:
Oh, I thought the dir block was closed with the first stage.  The way it is should be fine.